.github/workflows/ci.yaml

```diff
@@ -17,7 +17,18 @@ jobs:
           node-version: '16.x'
       - name: Install
         run: npm ci
+      - name: Lint Code
+        if: always()
```

Comment:
Are these `if`s needed?

Reply:
They let everything run even if one node fails. It'll still fail the check on a single failure